src/prefect/__init__.py

```diff
@@ -33,3 +34,5 @@
     _signal.signal(29, _sig_handler)
 except:
     pass
+
+context.logger = get_logger()
```

Comment:
Would you mind moving this to the `prefect.utilities.logging` module? The global logger is already created there, and the context is imported there as well. Makes more sense IMO to keep the logging stuff grouped together.
Could do it here:

https://github.com/PrefectHQ/prefect/blob/936c0e05f704f1fd6fa7da89d94734471418bd3e/src/prefect/utilities/logging.py#L251

```python
context.logger = prefect_logger = configure_logging()
```

Reply:
Moved the change as suggested.